packages/utils/src/eslint-utils/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from './applyDefault';
+export * from './context';
```

Comment:
This import was removed in https://github.com/typescript-eslint/typescript-eslint/pull/8377, but wasn't restored in https://github.com/typescript-eslint/typescript-eslint/pull/8399

This is a publicly visible change. Maybe we should fix it in separate PR?

Reply:
Probably yeah. The fact that nobody has reported anything makes me less motivated to - but, yeah, we should.

Reply:
Here it is:

https://github.com/typescript-eslint/typescript-eslint/issues/8975 -> https://github.com/typescript-eslint/typescript-eslint/pull/8976